NewsWeek web page clone

rukundoeric opened the pull request:
### What does this do? 
* Build Responsive Design with Bootstrap
### Description of Task to be completed? 
* A structural clone of  The NewsWeek web page
### How should this be manually tested? 
* Clone this repo  https://github.com/bini-i/Newsweek-Clone/tree/bootstrap-feature-branch
* Navigate to project directory.
* If you have a live server in you vs-code, just run it.
* Otherwise just open the index page with your browser.

bini-i committed:
Add index.html syle.css bootstrap README.md and github workflow

rukundoeric committed:
Add navbar

bini-i committed:
Update header list items

rukundoeric committed:
Add few styles on header-bottom

bini-i committed:
Add featured stories section of home page

bini-i committed:
Update Header layout and add bootsrap override class to body

rukundoeric committed:
Add top stories section & my turn section

bini-i committed:
Add right aside section structure and styling

rukundoeric committed:
Add few article items

rukundoeric committed:
Add editor's picks section

bini-i committed:
Add In megazine section structure and style

rukundoeric committed:
Merge conflict

rukundoeric committed:
Add Other News Section

bini-i committed:
add subscribe section structure and syling

rukundoeric committed:
Merge conflict

rukundoeric committed:
resolve stylelint errors

rukundoeric committed:
Resolve stylelint errors

rukundoeric committed:
Merge conflict

rukundoeric committed:
Merge conflict

bini-i committed:
Update header section and and layout of main row for width less than 992px

rukundoeric committed:
Add scroll effect

rukundoeric committed:
Merge branch 'bootstrap-feature-branch' of https://github.com/bini-i/Newsweek-Clone into bootstrap-feature-branch

rukundoeric committed:
Add scroll effect on navbar

bini-i committed:
Update README.md and index other news section medium screen layout

bini-i committed:
Update README.md

bini-i committed:
Update README.md

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1744 @@
+          </div>
+          <div class="d-flex align-items-center links">
+            <a class="login" href="#">LOGIN</a>
+            <a href="" class="mr-3 user"><i class="fas fa-user"></i></a>
+            <a href="" class="d-flex align-items-center subscribe-container">
```
use # as a path

arslanbisharat reviewed (changes requested):
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

 **Repo**

- [ ]Remove !important from CSS file 

- [x] Add something about your project in about section of your repo. This can help to understand features in your repo. 

- [x] Please use less CSS because the purpose of this project is to master bootstrap.

- [x] For href tag you can use the path **#**

- [x] Please make your repo more professional by adding all styling in one folder like screenshots, CSS, sass, images folder.  

**main**

- [x] On medium size, the text is overflowing. 
![chrome-capture (21)](https://user-images.githubusercontent.com/58504655/95324437-2523b080-08b9-11eb-9246-fd3755ff1d5a.jpg)


- [x] Here is an attachment from the original page. The images have more height than the text to the right. 
![chrome-capture (17)](https://user-images.githubusercontent.com/58504655/95323277-5a2f0380-08b7-11eb-920e-075a14cbb6bb.jpg)

- [x] You have copy-pasted the same code again and again. Please try to make it more attractive by changing images and using similar titles as presents on the original page. 
![chrome-capture (16)](https://user-images.githubusercontent.com/58504655/95323173-2bb12880-08b7-11eb-9a9c-4183db3430af.jpg)

- [x] You can have a look at the bottom attachment. at small size object Iphone5 there is white space but on the original page, this didn't happen. 
![chrome-capture (18)](https://user-images.githubusercontent.com/58504655/95323531-c3af1200-08b7-11eb-82f0-c5c034191ef3.jpg)


**Footer**
- [x]  Footer is disorganized at small objects.

- [x]  Please make footer links as [present on the original page](https://www.newsweek.com/). 
![chrome-capture (15)](https://user-images.githubusercontent.com/58504655/95322915-b5142b00-08b6-11eb-8968-8623dd8f9e8d.jpg)

- [x] Item are not align as present on the original page. Please make it as present on the original 
![chrome-capture (15)](https://user-images.githubusercontent.com/58504655/95322993-dbd26180-08b6-11eb-873b-caa68270d8a5.jpg)
page. 

**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

jubaan reviewed (changes requested):
# Status: Changes Required :recycle:

## :wave: :wave: Hello team!! Great job working on the previously required changes!

## Required Changes
### [Mandatory] :red_circle: 

- [x] Add to ```.gitignore``` files unrelated to the project, and nonproduction ready. The prettier config file is for local development, and SCSS uses the final CSS process file.
![image](https://user-images.githubusercontent.com/25774518/95325957-7d31c700-0867-11eb-885e-432f1bc58578.png)

- [x] MIT link is broken in README

- [x] Give all image ```alt="``` attribute an alternative text. Don't leave it blank.

- [x] Navbar is missing elements
![image](https://user-images.githubusercontent.com/25774518/95326894-e36b1980-0868-11eb-8a7c-e8dddc025494.png)

- [x] One element should be different from the others.
![image](https://user-images.githubusercontent.com/25774518/95327207-4ceb2800-0869-11eb-86b4-5bf0b332eb43.png)

- [x] Element does not match original in tablet view
![image](https://user-images.githubusercontent.com/25774518/95327408-9dfb1c00-0869-11eb-8fec-4917fba6d0fc.png)

- [x] Missing space between elements in subscribe section
![image](https://user-images.githubusercontent.com/25774518/95327592-e286b780-0869-11eb-981f-bf1a7053abb2.png)

- [x] Elemens don't match in newsletter section
![image](https://user-images.githubusercontent.com/25774518/95327646-fdf1c280-0869-11eb-8568-cd3950ce58e2.png)

- [x] The order of the sections on the mobile version doesn't match the actual page. 
**Original**
- top story
- featured stories
- The debate
- Opinion
- More Stories
- My turn

**Clone**
- **Features stories**
- **Top stories**
- **My turn**
- More stories
- the debate
- opinion

- [x] My turn section doesn't match original on the mobile version.
![image](https://user-images.githubusercontent.com/25774518/95327919-6771d100-086a-11eb-8b8c-23f9e2c50988.png)

- [x] Subscribe section doesn't look as original on the mobile version
![image](https://user-images.githubusercontent.com/25774518/95328598-642b1500-086b-11eb-947a-9b083fe92a11.png)

- [x] Newsletter section doesn't look as the original on the mobile version
![image](https://user-images.githubusercontent.com/25774518/95328646-77d67b80-086b-11eb-96b6-df4d49dd70e3.png)

- [x] Footer doesn't look like the original on the mobile version
![image](https://user-images.githubusercontent.com/25774518/95328709-9177c300-086b-11eb-97c2-e4590cee5777.png)

- [x] Navbar doesn't look like the original on the mobile version
![image](https://user-images.githubusercontent.com/25774518/95328835-ad7b6480-086b-11eb-931d-2f8073c15861.png)

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

bini-i committed:
Update file structure

bini-i committed:
Update footer layout alignment

rukundoeric committed:
Implement feedback

bini-i committed:
Update the subscribe section for mobil view

bini-i committed:
update linter requirement

jubaan reviewed (changes requested):
# Status: Changes Required :recycle:

## :wave: :wave: Hello!! Great job working on the previously required changes!

### [Mandatory] :red_circle: 

- [x] Elemens don't match in newsletter section
>   ![image](https://user-images.githubusercontent.com/25774518/95327646-fdf1c280-0869-11eb-8568-cd3950ce58e2.png)
- [x]  The order of the sections on the mobile version doesn't match the actual page.
>   **Original**
> * top story
> * featured stories
> * The debate
> * Opinion
> * More Stories
> * My turn
> 
> **Clone**
> 
> * **Features stories**
> * **Top stories**
> * **My turn**
> * More stories
> * the debate
> * opinion

- [x] Navbar is missing menu button in mobile
![image](https://user-images.githubusercontent.com/25774518/95354140-b54a0180-0889-11eb-8665-c564f6a1434d.png)

- [x] Newsletter section doesn't look as the original on the mobile version
>   ![image](https://user-images.githubusercontent.com/25774518/95328646-77d67b80-086b-11eb-96b6-df4d49dd70e3.png)

**Original**
![image](https://user-images.githubusercontent.com/25774518/95354370-f2ae8f00-0889-11eb-83f5-d90e05ccbb4a.png)

**Clone**
![image](https://user-images.githubusercontent.com/25774518/95354397-fcd08d80-0889-11eb-9370-52d35bb23241.png)

### [Comments] :green_circle: 
- A great tool to test if the project looks as intended in different screen sizes is using the browser media query inspector.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

bini-i committed:
Update order of HTML elements and layout of subscribe section for mobile view

bini-i committed:
Update for linters requriement

javitocor reviewed (changes requested):
# STATUS CHANGES REQUIRED ♻️ 

Great job guys with all the implemented changes! 👏 
Your project looks pretty nice! 💯 
Just a few adjustments to get it approved
Kindly, check my comments below 👇  apply the changes and resubmit your project for a new code review

## Mandatory Changes
- Kindly remove some CSS lines, this is a project to master bootstrap and you should use it everywhere it is available. You can display flex with d-flex, display none with d-none, etc... you do not need those lines in your CSS file, please implement everything you can using bootstrap classes.
- Screens below 768px:
   - The sections are not in the right order after opinion comes 'my turn' and then 'culture&travel'. Please modify it to make it equal to the original
   - In the magazine section, the first image should be bigger than the rest, kindly modify it
   - In the editor's picks section, next to the title should be the logo of the publication, now it has a photo, kindly remove it and add the logo
   - In the US, Business, sport.... sections, make the aside items pictures wider to look like the original
   - In the subscribe section, the images test and button must be aligned centre
   - in the footer, the social media icons should go under the Newsweek letters and copyright, please make it happen.

bini-i committed:
Fix Order of myturn and culture section
Fix magazine section, first image made larger
Fix editors section logo display
Fix US, Business, sport section picture made wider
Fix subscribe section, image made center
Fix footer section

rukundoeric committed:
Solve stylelint error

bini-i commented:
Fix Order of myturn and culture section …
Fix magazine section, first image made larger
Fix editors section logo display
Fix US, Business, sport section picture made wider
Fix subscribe section, image made center
Fix footer section social media order in mobile view

umairarshadbutt commented on assets/css/style.css:
```diff
@@ -0,0 +1,730 @@
+      display: none; } }
+
+.container {
+  padding: 0 !important;
```
Remove `!important`

umairarshadbutt commented on assets/css/style.css:
```diff
@@ -0,0 +1,730 @@
+  @media (max-width: 480px) {
+    .container {
+      max-width: 100% !important;
+      padding: 0 !important; } }
```
Remove `!important`

umairarshadbutt commented on assets/css/style.css:
```diff
@@ -0,0 +1,730 @@
+    .container {
+      max-width: 100% !important;
+      padding: 0 !important; } }
+  @media (max-width: 767px) {
```
Remove `!important`

umairarshadbutt reviewed (changes requested):
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly and work on the suggested changes  👇.
- [ ] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)


- **When you have made the specified changes, please submit another code review. Thank you!**


**If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

rukundoeric committed:
Implement feedback

rukundoeric commented:
Thanks, 

* `!Important` is removed everywhere in the code.

arslanbisharat reviewed (approved):
Congratslutons Approved 🎉
Hello Developer 👋
You are on 🔥
Thank you so much for making those changes and I am very happy to approve the project. You have done a great job. 👏
Please keep working hard and move forward with your curriculum.
Looking forward to seeing your next project. 😄
---------------------------------------------------------------------------------------------------------------
**To highlight**
- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat
